developer-docs-site/docs/nodes/ait/node-requirements.md

```diff
@@ -30,27 +30,28 @@ For running an Aptos **validator and fullnode** we recommend the following hardw
       - 2.8GHz, or faster
       - Intel Xeon Skylake or newer
   - **Memory**: 32GB RAM.
+  - **Storage**: 1T SSD with at least 40K IOPS and 200MiB/s bandwidth.
+  - **Networking bandwidth**: 1Gbps
 
 Example machine types on various clouds:
   - AWS
-      - c5.4xlarge
-      - c6i.4xlarge
+      - c6id.4xlarge (if use local SSD)
+      - c6i.8xlarge + io1/io2 EBS volume with 40K IOPS.
   - GCP
-      - c2-standard-16
-  - Azure
-      - Standard_B8ms
+      - n2-standard-16 (if use local SSD)
+      - n2-standard-32 + pd-ssd with 40K IOPS.
 
-## Storage requirements
+### Implications on hardware requirements
 
-The amount of data stored by the Aptos Blockhain depends on the ledger history (length) of the blockchain and the number of on-chain states (e.g., accounts). These values depend on several factors, including: the age of the blockchain, the average transaction rate and the configuration of the ledger pruner.
+The amount of data stored by the Aptos Blockhain depends on the ledger history (the number of transactions) of the blockchain and the number of on-chain states (e.g., accounts and resources). These values depend on several factors, including: the age of the blockchain, the average transaction rate, and the configuration of the ledger pruner.
 
-We recommend nodes have at least 300GB of disk space to ensure adequate storage space for load testing. You have the option to start with a smaller size and adjust based upon demands. You will be responsible for monitoring your node's disk usage and adjusting appropriately to ensure node uptime.
+Hardware requirements depend on the transaction rate and storage demands. Over time, hardware requirements will need to scale with these demands. The current hardware requirements are set with the consideration of estimated growth over the next 6 months.
 
-## Networking requirements
+Local SSD v.s. Network Storage
```

Comment:
Can you please change this to one of the below?
`**Local SSD vs. network storage**` OR
`### Local SSD vs. network storage` rest looks good. thanks!

Reply:
Done